tagmanager/src/tm_source_file.c

```diff
@@ -109,6 +109,35 @@ gchar *tm_get_real_path(const gchar *file_name)
 	return NULL;
 }
 
+/* add argument list of __init__() Python methods to the class tag */
+static void update_python_arglist(const TMTag *tag)
```

Comment:
Mmmwell, this is pretty specific to the Python parser and not well-contained, but well, probably still better than a whole new hook indeed.

BTW, this might be fixable in the parser side using the U-CTags' new Cork thingie? Not sure and not tested, but that would be nice.